lib/services/dbus.c

```diff
@@ -8,7 +8,6 @@
 #include <crm_internal.h>
 #include <crm/crm.h>
 #include <crm/services.h>
-#include <dbus/dbus.h>
```

Comment:
I think it's good practice to leave this in, since this file requires it, independently of pcmk-dbus.h's internal need for it.